client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -28,15 +29,15 @@ interface Props {
 	sites: Array< Site >;
 	onClose: () => void;
 	settings?: MonitorSettings;
-	monitorUserEmails?: Array< string >;
+	bulkUpdateSettings?: MonitorSettings;
```

Comment:
Both `setting` and `bulkUpdateSetting` is  **MonitorSetting** type. Is there a reason why we can't use `setting` alone?

Reply:
`settings` is for a single update, and `bulkUpdateSettings` is for bulk. We don't update all the fields in the `bulkUpdateSettings`.

Reply:
I hear what you mean.  I imagined something like letting the `NotificationSettings` figure out what fields to update based on what is being passed in the `settings`. But I guess that might add complexity.   

```
<NotificationSettings
   sites={ selectedSites }
   settings={ bulkUpdateSettings }
   ...
/>
```